From 9cfbfc4ca53bc0087112499cb58cc1eb5ae292f8 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Tue, 18 Oct 2022 12:29:15 +0300
Subject: [PATCH] [SW-0] package: arcboot: add .elf extenstion to elf file

To make ROM elf file self descriptive about the content of file
which is inturn copied to artifacts, this patch add .elf
extension to ROM elf file while copying to images folder.

Change-Id: I0855515fc4ab05ee2fa309be583a9708680cd800
---

diff --git a/package/arcboot/arcboot.mk b/package/arcboot/arcboot.mk
index 0f681b3..4aee372 100644
--- a/package/arcboot/arcboot.mk
+++ b/package/arcboot/arcboot.mk
@@ -17,7 +17,7 @@
 ARCBOOT_INSTALL_IMAGES = YES
 define ARCBOOT_INSTALL_IMAGES_CMDS
 	$(INSTALL) -D -m 0644 $(@D)/arcboot_rom.bin $(BINARIES_DIR)/arcboot_rom.bin
-	$(INSTALL) -D -m 0644 $(@D)/arcboot_rom $(BINARIES_DIR)/arcboot_rom
+	$(INSTALL) -D -m 0644 $(@D)/arcboot_rom $(BINARIES_DIR)/arcboot_rom.elf
 endef
 
 define HOST_ARCBOOT_INSTALL_CMDS
